src/CMakeLists.txt

```diff
@@ -293,6 +297,24 @@ target_link_libraries(openroad
   ${CMAKE_THREAD_LIBS_INIT}
 )
 
+add_library(libopenroad
+  ${OPENROAD_LIB_SRCS}
+  OpenRoad-py.cc
```

Comment:
curious if that could be a regular c++ library (i.e: drop the `OpenRoad-py.cc` dummy file) and have the main binary link against it?